### DISCUSSION TITLE
Comment from codeql scanner is rendered as nonsense link

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#56410] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub Mobile / Android

I was browsing:
https://github.com/github/codeql-action/pull/1694#pullrequestreview-1443536135

And saw:
![image](https://user-images.githubusercontent.com/2119212/240911602-e6545eb0-ed5d-4f38-9f1f-2bbb28892998.png)

The relevant content is:
```
[the enclosing function](1)
```

That _probably_ shouldn't be treated as a link, and it certainly isn't treated as a link by the web view:
<img width="922" alt="image" src="https://github.com/community/community/assets/2119212/018fc9f9-f901-4763-a906-e76cb60e5214">

Note that I'm not entirely certain about what I want and don't merely want these things to be actively disabled. My own action (which is why I'm there) takes advantage of the fact that I can generate link output to do so, but, at least for code review items, a reference to a number (any number) is almost certainly not something that should be treated as a link.

**Comment by jketema** [comment, marked as answer]
Thanks for reporting this. We'll try to get this fixed.

**Comment by Jamalbouzaidi** [comment]
Allô
